flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -169,6 +232,55 @@ public static void validateConsumerConfiguration(Properties config) {
 		}
 	}
 
+	public static void validateEFOConfiguration(Properties config, List<String> streams) {
+		if (config.containsKey(ConsumerConfigConstants.RECORD_PUBLISHER_TYPE)) {
```

Comment:
@xiaolong-sn Consider ["Avoid deep nesting of scopes, by flipping the if condition and exiting early."](https://flink.apache.org/contributing/code-style-and-quality-common.html)

Reply:
I've separate the method to a new one, so no `if` here.